man/fi_efa.7.md

```diff
@@ -66,11 +66,6 @@ The following features are supported:
   this bug would break those applications. This will be fixed in a future version
   of the EFA provider by adding proper support for *FI_PROGRESS_AUTO*.
 
-*Threading*
-: The RDM endpoint supports *FI_THREAD_SAFE*, the DGRAM endpoint supports
```

Comment:
We shouldn't change this? RDM endpoints should always support FI_THREAD_SAFE. I don't know if there is any application is using DGRAM endpoint + FI_THREAD_DOMAIN, but it's better to keep the default threading level unchanged to avoid breaking any. As we talked about, FI_THREAD_DOMAIN_RELAXED should be an opt-in